.github/workflows/zero-downtime-tests.yml

```diff
@@ -57,6 +57,11 @@ jobs:
       KUBECONFIG: ${{ matrix.KUBECONFIG }}
       KEPTN_NAMESPACE: "keptn-zd-test-${{ github.run_number }}-${{ github.run_attempt }}"
       GO_VERSION: "~1.18"
+      RUN_CONTINUOUS_DELIVERY_TEST: "true"
```

Comment:
is this variable still necessary, since we removed the CD test already?

Reply:
I'll remove it